site/_posts/2017-09-21-finalizers-can-be-interrupted-from-time-to-time.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Finalizers can be interrupted from time to time
+author: jrafanie
+date: 2017-09-21 15:36:39 UTC
+comments: true
+published: true
+tags: blog timeout tempfile finalizers
```

Comment:
I don't know if I did this part right.  😕

Reply:
Currently we use quite broad categories/tags: announcements, events, collaboration, releases, sprints, and LWIMIQ.

This can be tagged collaboration, or maybe it's about time we add a new tag: development